src/components/autosuggest/autosuggest.tsx

```diff
@@ -40,18 +39,31 @@ export const Autosuggest: React.FC<AutosuggestProps> = ({
       >
         {/* eslint-enable react/jsx-props-no-spreading */}
 
-        {originalData && status && isOpen && (
+        <AutosuggestText
+          textData={textData}
+          highlightedIndex={highlightedIndex}
+          getItemProps={getItemProps}
+        />
+        {textData.length > 0 && materialData.length > 0 && (
```

Comment:
I cannot overview all the functionality but isn't a check like:
```
textData && materialData && (...
```
enough?

Reply:
No because texData and materialData are defined no matter what. We have to check for content to make sure there is something to show.

Reply:
Ah it is arrays, ok :)